### PR TITLE
Adding QP's TINC test suites to CI pipeline

### PR DESCRIPTION
**Pull request by armenatzoglou**
In this PR, we add QP's TINC test suites to CI pipeline.
Based on the investigation in story #135663401, we do not need to add the workfile TINC test suite.

These changes have been tested and everything works fine.

### CODE REVIEW

**Comment by armenatzoglou** [comment]
@foyzur @hardikar @karthijrk @hsyuan Please take a look when you have some free time. Thanks

File: concourse/pipelines/pipeline.yml [hunk 1]
```diff
@@ -576,3 +576,83 @@ jobs:
...
+  - task: trigger_pulse
+    tags: ["gpdb5-pulse-worker"]
+    file: gpdb_src/ci/pulse/api/trigger_pulse.yml
+    input_mapping:
+      gpdb_src_archive: gpdb_src_tinc_tarball
+      installer_gpdb_rc: installer_rhel6_gpdb_rc
+      qautils_tarball: qautils_rhel6_tarball
+    params:
+      PULSE_URL: {{pulse_url}}
+      PULSE_PROJECT_NAME: "GPDB-5-CCLE"
```
Comment (ashwinstar) [inline concourse/pipelines/pipeline.yml]: Fix the project name, same below.

File: concourse/pipelines/pipeline.yml [hunk 2]
```diff
@@ -576,3 +576,83 @@ jobs:
       PULSE_PROJECT_NAME: "GPDB-5-CCLE"
       PULSE_USERNAME: {{pulse_username}}
       PULSE_PASSWORD: {{pulse_password}}
+
+- name: QP_memory-accounting
+  serial: true
```
Comment (ashwinstar) [inline concourse/pipelines/pipeline.yml]: Drop this no need to serialize the stuff.

File: concourse/pipelines/pipeline.yml [hunk 3]
```diff
@@ -576,3 +576,83 @@ jobs:
...
+  plan:
+  - aggregate:
+    - get: gpdb_src
+      tags: ["gpdb5-pulse-worker"]
```
Comment (ashwinstar) [inline concourse/pipelines/pipeline.yml]: Add `params: {submodules: none}` here to speed things a bit as no need to fetch submodule all it needs is concourse directory.

**Comment by foyzur** [comment]
Adding memory accounting changes that was part of a previous PR: https://github.com/greenplum-db/gpdb4/pull/200

This is necessary to push TINC tests. Without that fix the test  test_too_many_exec_accounts fails.

**Comment by foyzur** [comment]
Addressed all comments from @ashwinstar

**Comment by armenatzoglou** [comment]
Pushed to master 6f09d7242197def549888c9c3366d8fc9fcd99c6

**Pull request closed without merging**